libraries/SITL/SIM_SerialProximitySensor.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  Base class for serial proximity sensors
+*/
+
+#include "SIM_SerialProximitySensor.h"
+
+#include <AP_Math/AP_Math.h>
+
+#include <stdio.h>
+
+using namespace SITL;
+
+// uint16_t SerialProximitySensor::calculate_range_cm(float range_value) const
```

Comment:
probably best to remove this commented out code..

Reply:
Done.